bodhi/server/models.py

```diff
@@ -3845,24 +3847,24 @@ def unique_testcase_feedback(self):
         return filtered_feedbacks
 
     @property
-    def rss_title(self):
+    def rss_title(self) -> str:
         """
         Return a formatted title for the comment using update alias and comment id.
 
         Returns:
-            basestring: A string representation of the comment for RSS feed.
+            A string representation of the comment for RSS feed.
         """
         return "{} comment #{}".format(self.update.alias, self.id)
 
-    def __json__(self, *args, **kwargs):
+    def __json__(self, *args, **kwargs) -> str:
         """
         Return a JSON string representation of this comment.
 
         Args:
-            args (list): A list of extra args to pass on to :meth:`BodhiBase.__json__`.
-            kwargs (dict): Extra kwargs to pass on to :meth:`BodhiBase.__json__`.
+            args: A list of extra args to pass on to :meth:`BodhiBase.__json__`.
+            kwargs: Extra kwargs to pass on to :meth:`BodhiBase.__json__`.
         Returns:
-            basestring: A JSON representation of this comment.
+            A JSON representation of this comment.
```

Comment:
I think the previous docblock was actually incorrect here. Despite it being called ```__json__```, this actually returns a ```dict``` (strange, I know!)

```suggestion
            A JSON-serializable dict representation of this comment.
```